README.md

```diff
@@ -12,25 +12,44 @@ $ npm start
 Start Firefox in remote debugging mode and go to a tab you want to debug.
 
 ```
-$ /Applications/FirefoxNightly.app/Contents/MacOS/firefox-bin --start-debugger-server 6080
+npm run start-firefox
```

Comment:
I'd still like to have this command here. It's nice to be able to copy and paste for people that want to do it quickly and not create a config file. It also exposed exactly what `start-firefox` does.